bust-a-move-framework/src/main/java/bustamove/gamestate/DefeatScreen.java

```diff
@@ -89,8 +126,12 @@ public final void init(final GameContainer game,
         player2name = new Text("Player:", GameConfig.SIXT_LINE);
         player2name.centerText(game);
         player2score = new Text("Score:", GameConfig.SEVENTH_LINE);
-        player2score.centerText(game);
-        Button restart = new Button("Restart", GameConfig.FOURTH_LINE,
+        player2score.centerText(game);
+        player1highscore = new Text("", GameConfig.EIGTH_LINE);

```

Comment:
Initializing the text at the correct position, the text is later filled in with the score the player has.